src/modules/AccountSettings/hooks/useUpdateUser.ts

```diff
@@ -18,7 +18,7 @@ type MutationFnBody = {
   bodyToUpdate: UpdateUserBody
 }
 
-type ErrorResponse = {
+export type ErrorResponse = {
```

Comment:
Need to move to another dir where we can reuse easily.
Inside the modules is indicate that files only needed by the module itself.
Once another modules reuse the function, we need to move out the function.